core/builtins/message/__init__.py

```
@@ -184,7 +205,7 @@
         raise NotImplementedError
 
     async def finish(self,
-                     message_chain: Union[MessageChain, str, list, MessageElement] = None,
+                     message_chain: Optional[Union[MessageChain, str, list, MessageElement]] = None,
```

Ruff (0.8.2)

`MessageChain` may be undefined, or defined from star imports

(F405)